satgpt.py

```diff
@@ -92,14 +92,16 @@ def query_chatbot():
     try:
         # Parse the request data
         data = request.get_json()
-
+        print("here is the data in TRY ", data)
         # if query is not in data, return error
-        if "query" and "user_uuid" not in data and "r_hash" not in data:
+        if "query" not in data and "user_uuid" not in data and "r_hash" not in data:
             response = jsonify({"message": "No query provided"})
             return response, 400
 
         # check if we're in the middle of a payment
-        if "r_hash" in data:
+        elif "r_hash" in data:
+            #Is user credit checked here? 
```

Comment:
If you're asking questions in code, always mark the comment somehow prefixed with all caps so the reader knows the comment _really shouldn't_ be there once the question is answered. i.e. The standard "Delete me when I'm answered, done, or fixed" afaik are `FIXME` & `TODO` (this standardization makes them easily searchable if a reader is trying to find all the things they wanted to go back and fix/do), but you can use whatever you want here (i.e. `QUESTION`, `REVIEW`, etc)

```
# QUESTION: Is user credit checked here?
```

Then, once you're sure whether user credit is checked there or not, delete that comment and (if you feel it is needed), replace it with a descriptive comment that you think will help future readers of the code. If it is not needed, remove the comment entirely. This is really more of an art than a science, but we don't want to leave dead comments in there once they're solved & confuse our teammates!

Reply:
Ah ok, definitely sounds like a good documentation principle! I'll make sure to use FIXME or TODO in the future.